ocaml/xapi/helpers.ml

```diff
@@ -1312,3 +1312,16 @@ let retry ~__context ~doc ?(policy = Policy.standard) f =
 
 let retry_with_global_lock ~__context ~doc ?policy f =
   retry ~__context ~doc ?policy (fun () -> with_global_lock f)
+
+  (** [is_true] returns [true] if [key] in string map [assoc] denotes
```

Comment:
This `is_true` function is an exact duplicate of the function of the same name in `ocaml/xapi/vm_platform.ml` so please can we have just one of them?

Also we have a different `is_true` in `ocaml/xapi/xapi_db_upgrade.ml`
and our own versions of `bool_of_string` in `ocaml/xapi/cli_operations.ml` and `ocaml/xapi/system_domains.ml`
but we might want to keep those for their specific behaviours.